packages/components/psammead-consent-banner/src/index.stories.jsx

```diff
@@ -35,3 +43,38 @@ const props = {
 storiesOf('Components|ConsentBanner', module).add('default', () => (
   <ConsentBanner {...props} />
 ));
+
+storiesOf('Components|ConsentBannerWithKnobs', module)
+  .addDecorator(withKnobs)
+  .add(
+    'default',
+    inputProvider(
+      [
+        {
+          name: 'title',
+          defaultText: 'Privacy and Cookies Policy',
+        },
+        {
+          name: 'text',
+          defaultText: 'Changes to our Privacy and Cookie Policy ',
+        },
+        { name: 'acceptText', defaultText: 'OK' },
```

Comment:
Doesn't look like the `acceptText` or `rejectText` knobs are being used for anything - the `text` knob is being used 3 times instead